sites/next.skeleton.dev/src/examples/resources/cookbook/image-layouts/ExampleGrid.astro

```diff
@@ -0,0 +1,36 @@
+<!-- Flex -->
+<section class="flex justify-center">
+	<!-- Grid -->
+	<div class="grid grid-cols-2 gap-6 md:grid-cols-3">
+		<!-- Row -->
+		<div class="h-48 w-48">
+			<img class="h-full w-full rounded-lg object-cover" src="https://images.unsplash.com/photo-1617296538902-887900d9b592" alt="" />
```

Comment:
Per my previous note, let's reintroduce the Unsplash URL parameters to resize the images. Think of it like this:

- The params control the size of the image delivered from the CDN - which affects the file size - which affects the speed at which these load
- The Tailwind classes affect the display size.